libs/dmlf/include/dmlf/execution/execution_error_message.hpp

```diff
@@ -70,6 +70,11 @@ class ExecutionErrorMessage
     return message_;
   }
 
+  bool ok() const
```

Comment:
Would this read better?

```suggestion
  bool is_ok() const
```

Reply:
I think I'll go with `succeeded()`. what do you think?